content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,157 @@
+But how does it actually work?
+
+## The Basics
+
+Instead of a traditional record that contains music, a "control" record is used.
+Such a record contains a special signal that is processed by the computer.
+The computer transforms this signal into information about direction ("Is the record playing forwards or backwards?"), pitch ("How fast is the record spinning?") and position information ("At which position of the record is the needle?") and uses it to manipulate the playing track in software.
+
+There are many formats out there:
+[VirtualDJ](http://www.virtualdj.com/buy/controlvinyl.html), [Rekordbox](https://www.pioneerdj.com/product/features/software/rekordbox-dvs-control-vinyl/), [Numark](https://www.numark.com/product/virtualvinyl), [MixVibes](https://www.mixvibes.com/cross-dj-4/), [Traktor](https://www.native-instruments.com/en/products/traktor/digital-vinyl/traktor-scratch-a10/) and [Serato](https://serato.com/dj/pro/expansions/dvs?dvs=dvs-ready) all provide their own solution.
+Mixxx (and the underlying [xwax](https://xwax.org/) library) [support the latter 3 formats](https://manual.mixxx.org/2.3/en/chapters/vinyl_control.html#supported-timecode-media).
+
+Serato is one of the most popular and robust digital vinyl systems.
+The Serato Control CD can be used to archieve the same thing on CDJs instead of turntables.
+Since the latter essentially works the same as the vinyl version and can be [downloaded for free from the Serato Website](https://serato.com/controlcd/downloads), I'm going to explain how it works based on that format.
+
+Let's take a look at the control signal (also called "[timecode](https://en.wikipedia.org/wiki/Timecode) signal").
+If you open the WAV file with an audio editor like Audacity, it looks like this:
+
+![Serato CD Timecode Signal in Audacity]({static}/images/news/timecode-signal.png)
+
+As you can see, the timecode signal is a stereo signal.
+The left channel is on top and the right channel is on the bottom.
+Both channels look very similar, but the right channel is shifted a little.
+
+## Detecting the Playback Velocity
+
+Let's focus on the left channel for now.
+The wave basically looks similar to a sine wave that you may remember from your high school math classes.
+You can divide that signal into "cycles", where each cycle starts from zero and goes into a positive part (above the black line) followed by a negative part (below the black line) and then repeats:
+
+![Cycle of left channel in timecode signal]({static}/images/news/timecode-signal-cycle.png)
+
+Each cycle has the same length and if we count the number of cycles in a single second of audio, there are 1000 of them.
+Hence, we now know that the signal has a frequency of 1000 Hz (Hz = 1/s, so it's basically a fancy way saying "per second").
+
+With that information, the DVS system can detect how fast the record is playing by checking how many cycles there are in a second.
```

Comment:
somebody might be confused because its not clear that its counting the number of cycles in one second of **recorded audio**.